packages/concerto-core/lib/basemodelmanager.js

```diff
@@ -93,6 +93,16 @@ class BaseModelManager {
         this.options = options;
         this.addRootModel();
 
+        // TODO remove on full release of Map Type.
+        // Temporarily allow for ENABLE_MAP_TYPE environment variable to be set from options.
+        if (options?.flags) {
+            for (const [key, value] of Object.entries(options.flags)) {
+                if(key === 'ENABLE_MAP_TYPE') {
+                    process.env[key] = value;
```

Comment:
Why can't a client application do this?

Reply:
Some issues with Webpack from internal customer at DocuSign...

Reply:
I don't think this will work once the code is web packed by Babel. I think the code that is reading process.env.ENABLE_MAP_TYPE will have to be updated to reference an options object in the ModelManager to allow this value to be set dynamically by web clients.

Reply:
@dselman Matts concern is similar. I'll add a property based flag on the model manager to work around the issue.